gulpfile.js

```diff
@@ -140,10 +140,7 @@ gulp.task('examples', function() {
 });
 
 // Clean
-gulp.task('clean', function() {
-  return gulp.src(dest, {read: false})
-    .pipe(clean())
-});
+gulp.task('clean', require('del').bind(null, [dest]));
```

Comment:
Binding to null is clever but unreadable.
Would you mind to write it verbosely?